merlin/models/tf/outputs/topk.py

```diff
@@ -206,6 +206,8 @@ def call(
                 "You should call the `index` method first to " "set the _candidates index."
             )
 
+        if isinstance(inputs, tf.RaggedTensor):
```

Comment:
Thanks @sararb. I was trying to understand why we would have 1 as the 2nd dim of inputs, but from your explanation I now understand the reason is that we have predictions only for the last position.
I think it would be useful to add this remark as a comment.